engine/src/main/java/org/terasology/rendering/dag/nodes/DownSamplerForExposureNode.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.nodes;
+
+import org.terasology.assets.ResourceUrn;
+import org.terasology.config.Config;
+import org.terasology.config.RenderingConfig;
+import org.terasology.registry.In;
+import org.terasology.rendering.opengl.FBO;
+import org.terasology.rendering.opengl.FBOConfig;
+
+/**
+ * Extends the DownSamplerNode class purely to:
```

Comment:
As a tiny best practice note it is ideal to have a full sentence ended with a `.` here so it can parse out cleanly for the one-sentence summary used on some javadoc pages. In this case I dunno if we'd end up with something like:

`DownSamplerForExposureNode | Extends the DownSamplerNode class purely to:`

As the one liner explanation of the class in some places :-) Or maybe it would multi-line until it finds a . somewhere (in this case the whole thing)